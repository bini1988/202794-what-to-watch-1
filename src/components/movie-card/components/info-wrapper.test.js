import React from "react";
import renderer from "react-test-renderer";
import InfoWrapper from "./info-wrapper";

it(`InfoWrapper correctly renders default markup`, () => {
  const tree = renderer.create(
      <InfoWrapper className="new-class">
        {`Childs elements`}
      </InfoWrapper>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
it(`InfoWrapper correctly renders translate markup`, () => {
  const tree = renderer.create(
      <InfoWrapper className="new-class" translate>
        {`Childs elements`}
      </InfoWrapper>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
