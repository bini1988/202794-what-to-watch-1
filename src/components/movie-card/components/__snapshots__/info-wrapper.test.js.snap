// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoWrapper correctly renders default markup 1`] = `
<div
  className="movie-card__wrap"
>
  <div
    className="movie-card__info"
  >
    Childs elements
  </div>
</div>
`;

exports[`InfoWrapper correctly renders translate markup 1`] = `
<div
  className="movie-card__wrap movie-card__translate-top"
>
  <div
    className="movie-card__info"
  >
    Childs elements
  </div>
</div>
`;
